Makefile

```diff
@@ -131,3 +132,7 @@ runservices: my.env .docker-build
 .PHONY: stop
 stop: my.env
 	${DC} stop
+
+.PHONY: download
+download: my.env
+	${DC} run --rm --no-deps app shell make -f docker.make download
```

Comment:
As a side note, I've never liked the other make files. I think they're currently less makey and more hard-to-debug shell scripts. At some point, I want to look at redoing them.

This is fine for now, but if you feel like it was ornery (you can't run shellcheck on make files), then maybe we should split them into shell scripts in the `docker/` or `bin/` directories.

Reply:
let me see if I can convert ``docker.make`` quickly...

Reply:
The issues I saw in the first 30 minutes:

* It is easy to set `DATAMAPS_COMMIT` and `LIBMAXMIND_VERSION` in one place in `docker.make`, and then use it in multiple places. It's not clear where these should be set when refactoring to scripts.
* When building the Docker image, the Dockerfile copies ``docker.make`` to the image and runs it. When using scripts, should the scripts be copied one at a time, or should we copy the whole `docker` folder?

I'm guessing you have strong opinions on what these scripts should look like, and how they should be called. I'd rather review your refactor than try to implement it myself.